client/scss/components/_scroll_to_info.scss

```diff
@@ -0,0 +1,18 @@
+@keyframes scrollToWobble {
```

Comment:
Annoyingly I sat for ages thinking about why we would want to scroll to the wobbly line... Just a thought.